tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -92,51 +94,50 @@ sub run_test {
             #add the vf to the list of passthrough devices
             push @vfs, \%vf;
 
-            #plug the vf in guest
-            plugin_device($guest, $vfs[$i]);
+        }
 
-            #upload test specific logs
-            save_network_device_status_logs($log_dir, $guest, $i + 2 . "-after_hotplug_$vfs[$i]->{host_id}");
+        #hotplug the first vf to vm
+        plugin_device($guest, $vfs[0]);
```

Comment:
I prefer more specific function name if your plugin_device only deals with specific hardware.

Reply:
changed to "plugin_vf_device"